static-site/main.tf

```diff
@@ -0,0 +1,134 @@
+// S3
+resource "aws_s3_bucket" "mds_static_site" {
+  // bucket's name = domain name
+  bucket = "${var.sub_domain_name}"
+  acl    = "public-read"
```

Comment:
I think we don't want these to be public read.   We're going to be serving from Cloudfront using Origin Access Identity (OAI) instead.  It's fine to have it be public now if it's easier, but we'll be locking this down before we finish.